test/unit/server/services/mega/post-email-serializer.test.js

```diff
@@ -81,4 +84,91 @@ describe('Post Email Serializer', function () {
             output.plaintext.should.equal('hello');
         });
     });
+
+    describe('getTemplateSettings', function () {
+        before(function () {
+            models.init();
+        });
+
+        afterEach(function () {
+            sinon.restore();
+        });
+
+        it('uses the gloal settings when no newsletter_id is defined', async function () {
```

Comment:
Emails should always have a newsletter ID now, but we should fall back to the settings for the default newsletter. We'll be dropping the global settings in 5.0.